modules/node/src/onchainTransactions/onchainTransaction.service.ts

```diff
@@ -0,0 +1,34 @@
+import { CFCoreTypes } from "@connext/types";
+import { Injectable } from "@nestjs/common";
+import { TransactionResponse } from "ethers/providers";
+
+import { Channel } from "../channel/channel.entity";
+import { ConfigService } from "../config/config.service";
+
+import { OnchainTransactionRepository } from "./onchainTransaction.repository";
+
+@Injectable()
+export class OnchainTransactionService {
```

Comment:
we should have a `sendCollateral` as well right?

Reply:
We collateralize using the `deposit` function so we don't have/need a service level method for it.